docs/quick-install.md

```diff
@@ -1,16 +1,23 @@
 # Percona Everest quick install guide
 
-This section provides instructions for quickly installing and getting started with Percona Everest.
+Helm simplifies the installation of Percona Everest. With this guide, you'll be up and running with Percona Everest in no time. However, we also have a comprehensive [installation guide](install/install_everest_helm_charts.md) that covers all possibilities.
+
+Percona Helm charts can be found in [percona/percona-helm-charts](https://github.com/percona/percona-helm-charts) repository in Github.
+
```

Comment:
Should we instead link to the Everest Helm chart?

Reply:
Yes. Fixed